src/renderer/services/wallet/util.ts

```diff
@@ -17,3 +22,40 @@ export const hasKeystoreContent = (state: KeystoreState): boolean => isSome(getK
 export const hasImportedKeystore = (state: KeystoreState): boolean => isSome(state)
 
 export const isLocked = (state: KeystoreState): boolean => !hasImportedKeystore(state) || !hasKeystoreContent(state)
+
+export const assetWithBalanceMonoid = getMonoid<AssetWithBalance>()
```

Comment:
[suggestion] Could you move it ^ into `src/renderer/services/wallet/types.ts` where we have `type AssetWithBalance` defined?

Reply:
@veado yeah. i thought about it but there gonna be a semantic mismatch. imo at the `...types.ts` i suppose to see only types definitions. what do you think about it ?

Reply:
Agree, it's maybe not the best place there... How about to put it into `fpHelper` ? If yes, could you move [ClientStateM](https://github.com/thorchain/asgardex-electron/blob/06d09a1b5d7dc4029457917ae56361447ccad08a/src/renderer/services/types.ts#L14) (from a very previous PR) there as well, pls?

Reply:
im not sure that `fpHelpers` gonna be a good place for it too. We have `AssetWithBalance` only on terms of a `Wallet`. in case of moving it to app-level `fpHelpers` we will mix file structure dependencies and start to have a 'spaghetti' code 
as we mix dependency graph by importing smth from the down-side :(

Reply:
Let's make it simple and put it into `wallet/types.ts` as suggested earlier. So it's near to `AssetWithBalance` and `getMonoid` it still related to types (since `Monoid` is something on type level)

Reply:
ok. np